cpp/benchmarks/CMakeLists.txt

```diff
@@ -192,55 +192,62 @@ ConfigureBench(NULLMASK_BENCH "${NULLMASK_BENCH_SRC}")
 # - parquet writer chunks benchmark -----------------------------------------------------------------------------
 
 set(PARQUET_WRITER_CHUNKS_BENCH_SRC
-  "${CMAKE_CURRENT_SOURCE_DIR}/io/parquet/parquet_writer_chunks_benchmark.cpp")
+  "${CMAKE_CURRENT_SOURCE_DIR}/io/parquet/parquet_writer_chunks_benchmark.cpp"
+  "${CMAKE_CURRENT_SOURCE_DIR}/common/generate_benchmark_input.cpp")
```

Comment:
Should `generate_benchmark_input.cpp` be turned into a separate library and linked instead of including it in every test target?

Reply:
done, thanks for the suggestion